packages/server/api/src/app/app.ts

```diff
@@ -73,17 +69,7 @@ import { workerModule } from './workers/worker-module';
 export const setupApp = async (
   app: FastifyInstance,
 ): Promise<FastifyInstance> => {
-  const otelSDK = isLLMTelemetryEnabled()
-    ? new NodeSDK({
-        traceExporter: new LangfuseExporter({
-          secretKey: system.get(SharedSystemProp.LANGFUSE_SECRET_KEY),
-          publicKey: system.get(SharedSystemProp.LANGFUSE_PUBLIC_KEY),
-          baseUrl: system.get(SharedSystemProp.LANGFUSE_HOST),
-          environment: system.get(SharedSystemProp.ENVIRONMENT_NAME),
-        }) as SpanExporter,
-        instrumentations: [getNodeAutoInstrumentations()],
-      })
-    : undefined;
+  const otelSDK = getAiTelemetrySDK();
```

Comment:
I see that we also shut it down, should we do it for the engine too or is it not needed?